code/game/jobs/job/civilian.dm

```diff
@@ -18,14 +18,11 @@
 		if(4) H.equip_or_collect(new /obj/item/weapon/storage/backpack/satchel(H), slot_back)
 	H.equip_or_collect(new /obj/item/clothing/under/color/random(H), slot_w_uniform)
 	H.equip_or_collect(new /obj/item/clothing/shoes/black(H), slot_shoes)
-	if(H.backbag == 1)
-		H.equip_or_collect(new /obj/item/weapon/storage/box/survival(H), slot_r_hand)
-	else
-		H.equip_or_collect(new /obj/item/weapon/storage/box/survival(H.back), slot_in_backpack)

```

Comment:
you've removed a lot of instances of `H.back`, but other times you've left it in..why?